src/core/agent_space_interaction.jl

```diff
@@ -13,11 +13,7 @@ move_agent_single!, kill_agent!, coord2vertex, vertex2coord, genocide!
 Remove an agent from model, and from the space if the model has a space.
 """
 function kill_agent!(agent::AbstractAgent, model::ABM{A, S}) where {A, S<:AbstractSpace}
-  if typeof(agent.pos) <: Tuple
-    agentnode = coord2vertex(agent.pos, model)
-  else
-    agentnode = agent.pos
-  end
+  agentnode = coord2vertex(agent.pos, model)
```

Comment:
Agents on `GraphSpace`  still have integers for their positions, don't they? If so, this change will be wrong.

Reply:
This *may* be the case and if so, this was some overzealous cutting. I'm writing a test suite for these functions & will update the result whatever the outcome.

Reply:
This wasn't a problem here, but was indeed the case for `move_agent!`. Fixed with 
0cbc01f.